help_to_heat/portal/authentication_views.py

```diff
@@ -32,10 +32,21 @@ def _make_hash_value(self, user, timestamp):
 @require_http_methods(["GET", "POST"])
 class CustomLoginView(MethodDispatcher):
     template_name = "account/login.html"
-    error_message = "Something has gone wrong.  Please contact your team leader."
+    credentials_error_message = (
```

Comment:
Bit puzzled by this: why are these tuples, and why have you split the strings where you've split them?

Reply:
this was an IDE suggestion to stop any single line being too long, since theyre not comma seperated theyre not evalutated as tuples